src/ucs/arch/x86_64/cpu.c

```diff
@@ -19,6 +19,8 @@
 
 #define X86_CPUID_GENUINEINTEL    "GenuntelineI" /* GenuineIntel in magic notation */
 #define X86_CPUID_AUTHENTICAMD    "AuthcAMDenti" /* AuthenticAMD in magic notation */
+#define X86_CPUID_CENTAURHAULS    "CentaulsaurH" /* CentaurHauls in magic notation */
```

Comment:
Please add your organization copyright to the top of the file. The same relevant for the rest of updated files.

Reply:
fixed